charts/hail-search/values.yaml

```diff
@@ -34,30 +34,21 @@ environment:
   DATASETS_DIR: '/datasets'
   SSD_DATASETS_DIR: '/ssd-datasets'
 
+persistentVolume:
+  enabled: false
+  driver: ''
+  volumeHandle: ''
+
 initContainers: |-
   {{- range $path, $version := $.Values.global.hail_search.datasetVersions -}}
   {{ if $version }}
-  - name: mkdir-{{ $path | replace "/" "-" | replace "_" "-" | lower}}
-    image: busybox:1.35
-    command: ['/bin/mkdir', '-p', '{{ $.Values.environment.DATASETS_DIR }}/{{ $path | replace "SV" "SV_WGS" | replace "GCNV" "SV_WES"}}/annotations.ht']
-    volumeMounts:
-      - mountPath: {{ $.Values.environment.DATASETS_DIR }}
-        name: datasets
-  - name: mkdir-ssd-{{ $path | replace "/" "-" | replace "_" "-" | lower}}
+  {{- $localPath := $path | replace "SV" "SV_WGS" | replace "GCNV" "SV_WES" -}}
```

Comment:
The initialization action here is copying the annotations table from the mounted volume to the in-memory volume, once per reference genome/datasetType.